core/misc/tabledrag.js

```diff
@@ -633,6 +633,12 @@ Backdrop.tableDrag.prototype.pointerCoords = function (event) {
     return { x: event.pageX, y: event.pageY };
   }
 
+  // If pageX and pageY are defined in the event.originalEvent, use them instead
+  // of clientX and clientY to avoid potentially faulty pointer position calculations 
```

Comment:
Comments should not exceed the 80 character limit, and they should end with a period (the second line has an empty space instead):
```suggestion
  // If pageX and pageY are defined in the event.originalEvent, use them instead
  // of clientX and clientY to avoid potentially faulty pointer position
  // calculations.
```